django_app/redbox_app/redbox_core/client.py

```diff
@@ -67,3 +67,10 @@ def rag_chat(self, message_history: list[dict[str, str]], token: str) -> SimpleN
         logger.debug("response_data: %s", response_data)
 
         return response_data
+
+    def get_file_status(self, file_id: uuid, token: str) -> SimpleNamespace:
```

Comment:
Other client methods take the User object and pull the token out of it rather than just passing the token. It might be worth being consistent.

Reply:
Yeah I'm going to raise a ticket for tidying up the client, it could do with a bit of refinement